_docs/concepts/auth.md

```diff
@@ -0,0 +1,138 @@
+---
+category: Concepts
+title: Auth
+overview: Architectural deep-dive into the design of Auth, which provides the secure communication channel and strong identity for Istio.
+              
+parent: Network and Auth
+order: 20
+
+bodyclass: docs
+layout: docs
+type: markdown
+---
+
+The page explains Auth's role and general architecture.
+
+
+## Istio Auth Concept
+
+*Status: Draft*
+
+*Date: April, 2017*
+
+
+[TOC]
+
+
+### Overview {#overview}
+
+Istio Auth aims at enhancing the security of microservices and their communication without requiring service code changes. It is responsible for:
+
+
+
+*   Presenting a strong identity that represents the role of the service to enable interoperability across clusters and clouds.
+*   Securing the service to service communication and the end-user to service communication.
+*   Providing a key management system to automate key/cert generation, distribution, rotation, and revocation.
+*   Upcoming features:
+    *   fine-grained authorization and auditing to control and monitor who accesses your services, apis, or resources, 
+    *   powerful authorization mechanisms: [ABAC](https://en.wikipedia.org/wiki/Attribute-Based_Access_Control), [RBAC](https://en.wikipedia.org/wiki/Role-based_access_control), Authorization hooks.
+
+### Architecture {#architecture}
+
+Figure 1 shows the Istio Auth architecture, which includes three important components: identity, key management, and communication security. This diagram describes how Istio Auth is used to secure the service-to-service communication between service A running as the service account "foo" and service B running as the service account "bar".
```

Comment:
This diagram describes how Istio Auth is used to secure the service-to-service communication between service A, running as the service account "foo", and service B, running as the service account "bar".